include/odp/api/spec/timer.h

```diff
@@ -137,9 +141,28 @@ odp_timer_pool_t odp_timer_pool_create(const char *name, const odp_timer_pool_pa
  * The purpose of this call is to coordinate the creation of multiple timer
  * pools that may use the same underlying HW resources.
  * This function may be called multiple times.
+ *
+ * @deprecated Use odp_timer_pool_start_multi() instead
  */
 void odp_timer_pool_start(void);
 
+/**
+ * Start timer pools
+ *
+ * Start given timer pools. After a pool has been successfully started the pool handle can be used
+ * with other APIs. Each timer pool can be started only once.
+ *
+ * Returns 'num' when all given timer pools have been successfully started. If the return value N is
+ * less than 'num', only the first N timer pools have been started and the rest failed to start.
```

Comment:
When N < num, at least some of the rest timer pools failed to start and those cannot be used anymore (can be only destroyed).

Reply:
Updated in V2.